deepmd/pt/model/model/make_hessian_model.py

```diff
@@ -115,8 +115,8 @@ def forward_common(
 
         def _cal_hessian_all(
             self,
-            coord: torch.Tensor,
-            atype: torch.Tensor,
+            coord,
+            atype,
```

Comment:
why remove this

Reply:
For consistency with lines in `make_model.py`; now recovered.